lib/Mojo/Util.pm

```diff
@@ -792,7 +792,7 @@ Create anonymous scope guard object that will execute the passed callback when t
 
   my $bool = secure_compare $str1, $str2;
 
-Constant time comparison algorithm to prevent timing attacks.
+Constant time comparison algorithm to prevent timing attacks. The secret string should be the second argument, to avoid leaking information about the length of the string.
```

Comment:
The line is longer than 120 characters now.

Reply:
Ok. Fixed. 

I normally keep lines under 80 characters but wrongly assumed because of long lines for other sections that you weren't wrapping lines.  I think a style guide would be useful, or (better yet) add a POD test that enforces style.

Reply:
The practical guide is that there is a perltidy config in the repo which should be used for all contributions and code/doc styles in use should be adhered to when appropriate.